modules/management-console-agent/src/main/java/org/apache/ignite/agent/StompDestinationsUtils.java

```diff
@@ -92,20 +92,18 @@ public static String buildEventsDest(UUID clusterId) {
 
     /**
      * @param clusterId Cluster ID.
-     * @param reqId request ID.
      * @return Task response destination.
      */
-    public static String buildActionTaskResponseDest(UUID clusterId, UUID reqId) {
-        return CLUSTER_ACTION_TASK_RESPONSE_DEST + clusterId + "/" + reqId;
```

Comment:
I think we should not change topic name, because i don't sure how it will be handled in multi-backend setup :(

Reply:
Same way. Only one backend receive message.